apps/expo/src/components/AnnouncementCard.tsx

```diff
@@ -0,0 +1,65 @@
+import { Image, Text, View } from "react-native";
+import type { ImageSourcePropType } from "react-native";
+
+import type { RouterOutputs } from "~/utils/api";
+
+type GetAnnouncementOutput =
+  RouterOutputs["announcement"]["getAnnouncements"][number];
+
+interface AnnouncementCardProps {
+  announcement: GetAnnouncementOutput;
+}
+
+const AnnouncementCard: React.FC<AnnouncementCardProps> = ({
+  announcement: { title, createdAt },
+}) => {
+  const createdBy = "Michelle C.";
+  const userImageSrc: ImageSourcePropType = {
+    uri: "https://csncollision.com/wp-content/uploads/2019/10/placeholder-circle.png",
+  };
+  const titleImageSrc: ImageSourcePropType = {
+    uri: "https://s3-alpha-sig.figma.com/img/f246/82fa/befee9f4978b6cc8b351a2f3b81d456d?Expires=1706486400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=fPv06M5fmTfRNUsxZoeLLUUsdCK2ISw6gt6qir1vkJp-xIWN6mKFegXWCWTKTfFvQYHMPPCxKRT7-vLH0NkKPm~Nj2s~AFwHgXunypaS0nhLeecpbEBMBSysfkwFRjN5vK-dUSiGArjau-DQ6v262W6bCg-4L85Ykj5-4Ii0Bu0scVsNjSSRkM1r00OzJChy-hpgsHGuT-V3DwT0Pta~Hs2KT44TM3p1IK3tIgIZuvc6hoQJXmPCctSm1EV1Oxhs~hARLqbkUtVDGxFbUAoew0xjRC1B1Xc7qZEB-v0HFYEC0gTv9SQx9GdswPtJYFvb1BgVl1alkaWpV7ZGTJujWA__",
+  };
+
+  const createdAtFormatted = new Intl.DateTimeFormat("en-US", {
+    year: "2-digit",
+    month: "2-digit",
+    day: "2-digit",
+  }).format(createdAt);
+
+  return (
+    <View className="border-nv-80 flex flex-row items-center justify-start rounded-xl border py-[10px]">
+      <View className="flex-[0.3] items-center">
+        <Image
+          source={titleImageSrc}
+          accessibilityLabel={`Announcement title image`}
+          className="h-14 w-14"
+          resizeMode="cover"
+        />
+      </View>
+      <View className="flex-[0.7] gap-4 p-4 pl-0">
+        <Text
+          numberOfLines={1}
+          className="text-base font-medium tracking-[0.15px]"
+        >
+          {title}
+        </Text>
```

Comment:
right now, the title gets cut off after the first line. Can we make it so that it can expand to maybe 2 or 3 lines if the title is long enough?

Reply:
Sure, I can do that. But it wouldn't look like the figma design, is that ok?
![image](https://github.com/uoftblueprint/centre-for-dreams/assets/92739966/c261ca3c-c453-4487-bbd0-9d666d8f5fb8)

Reply:
you can keep it how it is for now then -- we can bring this up with the designers later